src/base/utils/string.cpp

```diff
@@ -27,124 +27,74 @@
  * exception statement from your version.
  */
 
-#include <QByteArray>
-#include <QString>
-#include <QLocale>
-#include <cmath>
 #include "string.h"
 
-QString Utils::String::fromStdString(const std::string &str)
-{
-    return QString::fromUtf8(str.c_str());
-}
+#include <cmath>
 
-std::string Utils::String::toStdString(const QString &str)
-{
-    QByteArray utf8 = str.toUtf8();
-    return std::string(utf8.constData(), utf8.length());
-}
+#include <QByteArray>
+#include <QtGlobal>
+#include <QLocale>
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 2, 0))
```

Comment:
I remember a user complaining about compilation broken because he used Qt 5 but version < 5.2.
`QBT_USES_QT5` now define as `Qt major version > 4` maybe we should bump that?